### PR TITLE
Dirty hack, fix the permission of direcotry /var/log/xcat

### PR DESCRIPTION
**Pull request by neo954**
On Ubuntu, the ``rsyslogd`` daemon write log files with ``syslog:adm`` permissison. And in some case, the directory ``/var/log/xcat`` was created by xCAT, and had ``root:root`` ownership. In this way, ``rsyslogd`` did not have enough permission to write to log files under this directory.

As a dirty hack, change the ownership of directory ``/var/log/xcat`` to the same ownership of directory ``/var/log``.

### CODE REVIEW

**Comment by immarvin** [comment]
hi @neo954 , I have 2 comments:
1. the ``chown`` logic should be placed in ``syslog`` postscript instead of the spec file
2. maybe we can consider to use ``chown --reference`` 
````
       --reference=RFILE
              use RFILE's owner and group rather than specifying OWNER:GROUP values
````

File: xCAT/debian/postinst [hunk 1]
```diff
@@ -20,9 +20,9 @@ set -e
 
 case "$1" in
     configure)
-         #make dir for the current pid
-         mkdir -p /var/run/xcat
-	 cp -f /opt/xcat/share/xcat/scripts/xHRM /install/postscripts/
+        # Make dir for the current pid
+        mkdir -p /var/run/xcat
+	cp -f /opt/xcat/share/xcat/scripts/xHRM /install/postscripts/
```
Comment (samveen) [inline xCAT/debian/postinst]: Is the indentation still off?

File: xCAT/debian/postinst [hunk 2]
```diff
@@ -41,11 +41,23 @@ case "$1" in
 	    xcatconfig -i -d -s
 	fi
 
-#        [ -e /etc/apache2/conf-enabled/xcat.conf ] && rm /etc/apache2/conf-enabled/xcat.conf
-         ln -s -f /etc/apache2/conf-available/xcat.conf.apach24 /etc/apache2/conf-enabled/xcat.conf
+        ln -s -f /etc/apache2/conf-available/xcat.conf.apach24 /etc/apache2/conf-enabled/xcat.conf
```
Comment (samveen) [inline xCAT/debian/postinst]: Here too?

File: xCAT/debian/postinst [hunk 3]
```diff
@@ -41,11 +41,23 @@ case "$1" in
...
 
 	/etc/init.d/apache2 restart
 
+        # On Ubuntu, the rsyslogd daemon write log files with syslog:adm
```
Comment (samveen) [inline xCAT/debian/postinst]: Here onwards as well, by the looks of it.

File: xCAT/debian/postinst (hunk 1 above)
Comment (whowutwut) [inline xCAT/debian/postinst]: This line is still off in the indentation, it's a tab where the others are spaces.

Comment (whowutwut) [inline xCAT/debian/postinst]: Line 26-28 and 39-42 also tabs..

**Pull request merged**